x-pack/plugins/maps/public/store/ui.js

```diff
@@ -65,9 +69,16 @@ export function enableFullScreen() {
     isFullScreen: true
   };
 }
+export function setReadOnly(isReadOnly) {
```

Comment:
I'd remove this from this PR. IT's basically unused code. Introduce these action/action-creator/reducer when we add the setting of this in dashboard-mode.

(keep the selector of course).